src/components/App.js

```diff
@@ -127,7 +132,9 @@ class App extends Component {
           />
         )}
         {this.state.renderCompleted && <Completed todo={this.state.todoList} />}
-        {this.state.renderIncomplete && <Incomplete />}
+        {this.state.renderIncomplete && (
```

Comment:
Variable name `renderIncomplete` does not seems like boolean.. use instead isIncompleteTodoVisible